tox.ini

```diff
@@ -5,15 +5,16 @@ small_test_ee = quay.io/ansible/python-base
 mypy_tmp_dir = mypy_tmp_dir
 
 [tox]
-envlist = linters, type, py38, report, clean
-minversion = 3.16.1
+envlist = linters, type, py{36,37,38,39,310}, report, clean
+# we do not want debug old bugs
+minversion = 3.24.3
 skipsdist = True
 skip_missing_interpreters = true
 
 
 [testenv]
 description = Run pytest under {basepython} ({envpython})
-# the pytest command line is not in the project.toml because of issues 
+# the pytest command line is not in the project.toml because of issues
```

Comment:
This does not seem related.

Reply:
I added it to comment, and I hope you do not suggest to make a standalone change to remove that extra space.

Reply:
I would prefer it to be separate but I won't block. I was reviewing this from the GH Android app and it wasn't even visible what changed here, they don't illuminate those spaces.

Reply:
I just realized that it also has a typo in "pyproject". This is one of the reasons I prefer separate PR — small mistakes like this will block the entirety of the pull request, including perfectly good changes that could've been merged ages ago.